pydatastructs/miscellaneous_data_structures/disjoint_set.py

```diff
@@ -124,3 +124,24 @@ def find_size(self, key):
             raise KeyError("Invalid key, %s"%(key))
 
         return self.find_root(key).size
+
+    def groups(self):
```

Comment:
By `groups` you mean returning a list of disjoint sets formed so far in the data structure?

Reply:
If so I'd say use `sets` because they are actually sets.